src/main/scala/chisel3/aop/Select.scala

```diff
@@ -392,7 +392,7 @@ object Select {
         .block
         .getCommands()
     ) {
-      case Attach(_, seq) if seq.contains(signal) => seq
+      case Attach(_, seq) if seq.contains(Node(signal)) => seq
```

Comment:
The type of `seq` is `Seq[Node]` while `signal` is `Data` and obviously `Data` can never equal `Node`.